apps/back-end/test/data/datasets/some-file/items/0.json

```diff
@@ -0,0 +1 @@
+{"id":0, "a":"a:foo", "b":"b:bar", "desc": "humbug"}
```

Comment:
Whilst we're changing these file names, I'm thinking the folder name `some-file` could also be changed to something like `dummy-dataset` to make the test data more instructive as an example.

Reply:
I can change that if you like.

Reply:
I'm now seeing no test data in https://github.com/DigitalCommons/mykomap-monolith/tree/ts-restify-the-backend/apps/back-end/test

Reply:
The test data *should* still be in `apps/back-end/test/data/datasets`. But I see it isn't... investigating and fixing.

Reply:
Ok, just an accident - the new versions weren't staged in the commit. Fixed in an appended fixup commit for later integration.